### ISSUE TITLE
Visual bug in the form block

### ISSUE THREAD

**Issue by RuanMD** [baptisteArno/typebot.io#307]
![image](https://user-images.githubusercontent.com/44640884/218146806-8fb9863f-782c-45f9-bbc0-091a966f73c4.png)

**Comment by baptisteArno** [comment]
Yes, it sometimes happens on flows with lots of blocks. Moving the concerned group re-position the arrow.

I will at some point see what I can do, but that's an issue with rendering optimization.
